
using device: cuda:0
Traceback (most recent call last):
  File "sac_continuous_action.py", line 445, in <module>
    envs = task_registry.make_task(task_name=args.task)
  File "/home/shruti/workspaces/aerial_gym_ws/src/aerial_gym_simulator/aerial_gym/registry/task_registry.py", line 26, in make_task
    task_class = self.get_task_class(task_name)
  File "/home/shruti/workspaces/aerial_gym_ws/src/aerial_gym_simulator/aerial_gym/registry/task_registry.py", line 11, in get_task_class
    return self.task_class_registry[task_name]
KeyError: 'reach_avoid_task'
Traceback (most recent call last):
  File "sac_continuous_action.py", line 445, in <module>
    envs = task_registry.make_task(task_name=args.task)
  File "/home/shruti/workspaces/aerial_gym_ws/src/aerial_gym_simulator/aerial_gym/registry/task_registry.py", line 26, in make_task
    task_class = self.get_task_class(task_name)
  File "/home/shruti/workspaces/aerial_gym_ws/src/aerial_gym_simulator/aerial_gym/registry/task_registry.py", line 11, in get_task_class
    return self.task_class_registry[task_name]
KeyError: 'reach_avoid_task'